modules/bezug_kostalplenticoreem300haus/main.sh

```diff
@@ -10,9 +10,6 @@ else
 	MYLOGFILE="${RAMDISKDIR}/evu.log"
 fi
 
-bash "$OPENWBBASEDIR/packages/legacy_run.sh" "bezug_kostalplenticoreem300haus.kostal_plenticore" "${kostalplenticorehaus}" >>"${MYLOGFILE}" 2>&1
-ret=$?
-
-openwbDebugLog ${DMOD} 2 "RET: ${ret}"
+bash "$OPENWBBASEDIR/packages/legacy_run.sh" "modules.kostal_plenticore.device" "counter" "${kostalplenticoreip}" "" "0" "" "${kostalplenticorehaus}" >> "$MYLOGFILE" 2>&1
```

Comment:
Du hast die fehlenden Parameter jetzt hinzugefügt. Ist natrürlich auch eine Möglichkeit. Wobei du dann meines Erachten `"none"` statt `""` verwenden. Einfacher fände ich es allerdings, wenn du die ungenutzten Parameter in Python entfernst.